lib/flight_job/outputs/info_script.rb

```diff
@@ -51,7 +53,13 @@ def register_all
         Time.parse script.created_at
       end
 
-      register(section: :notes, header: 'Notes') { |s| s.notes }
+      if humanize?
+        notes = MarkdownRenderer.new(object.notes).wrap_markdown
+      else
+        notes = object.notes
+      end
+      register(section: :notes, header: 'Notes') { notes }
```

Comment:
It's worth noting that this method of first branching to determine how the notes should look and *then* registering the column works fine when a single object is being displayed such as an `info-*` command.  However, this would not work for a `list-*` command which is rendering multiple objects.

The code is fine as it is, but its probably worth making a mental note of the following.

For a `list-*` command the following would be needed:

```ruby
register(section: :notes, header: 'Notes') do |s|
  if humanize?
    MarkdownRenderer.new(s.notes).wrap_markdown
  else
    s.notes
  end
end
```

The block provided to `register` is called once for each item in the list and each item (with the above) would use its own notes.  With the way that you have it, the same set of notes would be used for each item in the list.

(The above will also work just fine for an `info-*` command).